.pylintrc

```diff
@@ -0,0 +1,6 @@
+[MESSAGE CONTROL]
+disable = abstract-method,dangerous-default-value,invalid-all-object,lost-exception,missing-docstring,syntax-error
```

Comment:
`syntax-error` doesn't sound good!

Reply:
No it is not! now removed. Was added when I was testing an alternative mock and the mock lib was throwing some strange syntax errors.